modules/gltf/test/lib/gltf-scenegraph-modifiers.spec.js

```diff
@@ -37,3 +37,71 @@ test('GLTFScenegraph#addImage', t => {
 
   t.end();
 });
+<<<<<<< HEAD
```

Comment:
Merge conflict. Does this file need to be changed?

Reply:
So that is why I created this PR. @Avnerus What do you think? Should we remove this test case for **2.3-release?**

Reply:
Well it looks like that minmax calculation doesn't exist on the `2.3-release` branch, so my guess is that the test could be removed.